CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 13.3.0 (08/01/23)
+
+* Added azure support [#29734](https://github.com/gravitational/teleport/pull/29734)
+* Fixed Kubernetes Legacy Proxy heartbeats [#29738](https://github.com/gravitational/teleport/pull/29738)
+* Added Postgres backend and Azure session storage [#29705](https://github.com/gravitational/teleport/pull/29705)
+* Bumped Buf to v1.25.0 [#29701](https://github.com/gravitational/teleport/pull/29701)
```

Comment:
I'm in 2 minds about including dependency updates. On one hand our changelogs have been feature-oriented. But on the other, I can also see that someone watching for fixes would still be interested in this.

Reply:
I excluded the buf update because that seemed to not matter as a rather dev-oriented thing. I did include the Go toolchain update because I noted it was previously in changelogs, perhaps because security conscious users may want to know we are not using a vulnerable version of the toolchain/libraries.

Reply:
👍 Go or Rust updates make a lot of sense. I wouldn't include buf updates.

Reply:
Thanks all. Comments make sense and I agree. I'll remove the line.